Sources/BraveNews/Composer/FeedDataSource.swift

```diff
@@ -108,10 +108,13 @@ public class FeedDataSource: ObservableObject {
 
   private let todayQueue = DispatchQueue(label: "com.brave.today")
   private let reloadQueue = DispatchQueue(label: "com.brave.today.reload")
+  private weak var historyAPI: BraveHistoryAPI?
```

Comment:
Why would this be `weak`?

Reply:
Fix at https://github.com/brave/brave-ios/pull/7087/commits/7f0d662a53231d133f4d69b9fee05c97c988603a